apps/web/components/dialog/ReassignDialog.tsx

```diff
@@ -135,21 +135,24 @@ export const ReassignDialog = ({ isOpenDialog, setIsOpenDialog, teamId, bookingI
     },
   });
 
-  const [showConfirmation, setShowConfirmation] = useState(false);
+  const [confirmationModal, setConfirmationModal] = useState<{
+    show: boolean;
+    membersStatus: "unavailable" | "available" | null;
```

Comment:
Can we simplify this maybe to "isAvailable"? Feels like a bit complex to have an enum type.